datafusion/core/benches/jit.rs

```diff
@@ -23,9 +23,9 @@ extern crate datafusion;
 mod data_utils;
 use crate::criterion::Criterion;
 use crate::data_utils::{create_record_batches, create_schema};
-use datafusion::row::jit::writer::bench_write_batch_jit;
```

Comment:
we could re-export the new crate from datafusion so we don't have to change these imports - we are doing this for the other datafusion crates:

``` rust
// re-export DataFusion crates
pub use datafusion_common as common;
pub use datafusion_data_access;
pub use datafusion_expr as logical_expr;
pub use datafusion_physical_expr as physical_expr;
```
```

Reply:
Thanks for pointing this out! fixed